README.md

```diff
@@ -11,37 +11,77 @@ intuitive, and predictable.
 
 ```console
 admin              Administrative commands
-alloc              allocate a new instance for interactive use
+alloc              Interactively enqueue a job within a sub allocation
 attach             Interactively attach to job
+batch              Non-interactively enqueue a job within a sub allocation
 cancel             Cancel one or more jobs
-info               Display info for a job
-kill               Send signal to one or more running jobs
+info               Display info for one or more jobs
+kill               Forcefully kill one or more running jobs
 signal             Send a signal (e.g., raise exception) on one or more jobs
 cron               Schedule tasks on timers and events
-exec               Execute processes across flux ranks
-jobs               flux jobs queue listing
-purge              Purge the oldest inactive jobs
-resource           list/manipulate Flux resource status
-run                run a job interactively
+exec               Execute processes across flux ranks in a job
+jobs               Show queued, running, and historical flux jobs
+prune              Purge the oldest inactive jobs
+run                Run a command as a job interactively
 ssh                Create proxy environment for Flux instance
-submit             enqueue a job
-system             list/manipulate Flux resource status
+submit             Non-interactively enqueue a job
+system             System commands
 version            Display flux version information
-wait               Wait for job(s) to complete.   
+wait               Wait for job(s) to complete  
 ```
 
 ### flux submit
 
-Submit flux jobs
+#### Interactively
+
+Run a flux job in an existing flux allocation
+```console
+# Submit a job directly to get back an id
+$ flux run sleep 10
+ƒhejTu4KTFm
+
+# Submit a flux job file
+$ flux run -f mybatch.sh
+ƒhejTu4KTFm
+```
+
+Run a flux job in a new flux allocation
+
+```console
+# Submit a job directly to get back an id
+$ flux alloc sleep 10
+ƒhejTu4KTFm
+
+# Submit a flux job file
+$ flux alloc -f mybatch.sh
+ƒhejTu4KTFm
+```
+
+#### Non-interactively
+
+Submit flux jobs to an existing flux allocation
```

Comment:
This isn't necessarily an allocation - I would just put here "Submit jobs to a flux instance." I think instance is the term you are looking for.

Reply:
So you would be okay with the following?

```
Submit jobs within an existing flux instance.
```

I think the problem here is that flux doesn't have a well defined vocabulary for the hierarchical partitioning you can perform by creating your own flux instance within an allocation. We could call them partitions but I haven't seen that elsewhere. E.g. we need a way to say I'm running this cmd in this allocation from the workload manager or I'm running this cmd in a new flux instance that's running within the allocation from the global flux instance. Aka how do we show the recursive nature of flux in language so that that it's easily understandable to users. 

`flux mini batch` and `flux mini alloc` --> execute jobs one layer down in the flux tree
`flux mini submit` and flux mini run` --> execute jobs at the current layer in the flux tree

Reply:
Yep. And agree, I wouldn't use "partition" because that means something else in HPC. We started a glossy to hopefully have better definitions for things moving forward. https://flux-framework.readthedocs.io/en/latest/glossary.html. Generally:

- a flux instance is flux running on some set of resources, and the level of the graph is unknown (e.g., I have a slide in my talk that shows this)

![image](https://user-images.githubusercontent.com/814322/220199403-db82b538-d0b9-42f2-b151-75a952448f34.png)

We _assume_ that a user is submitting to a top level, but it's not necessarily the case.

> E.g. we need a way to say I'm running this cmd in this allocation from the workload manager or I'm running this cmd in a new flux instance that's running within the allocation from the global flux instance. 

My 0.02 is that pictures would really help here, we can eventually add them to that docs page / tutorials to explain concepts.